week2/assignment/week2.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-, initial-scale=1.0">
+    <link rel="stylesheet" href="week2.css">
+    <title>week2 YoonjiLee</title>
+</head>
+<body>
+    <header class="page">
+        <div class="left-side">
+            <img src="./img/bars.svg">
+        </div>
+        <div class="title">
+            <h1>회원 관리 페이지</h1>
+        </div>
+        <div class="right-side">
+            <h2>AND SOPT 웹 YB 이윤지</h2>
+        </div>
+    </header>
+
+    <main>
+        <form class="filter">
+            <div class="name">
+            <label for="name">이름</label>
+            <input type="text" id="name" placeholder="이름을 입력하세요"/>
+            </div>
```

Comment:
```suggestion
            <div class="name">
                <label for="name">이름</label>
                <input type="text" id="name" placeholder="이름을 입력하세요"/>
            </div>
```
들여쓰기 해주세요! 들여쓰기를 통해 가독성과 더불어 부모-관계를 쉽게 파악할 수 있어요.
에디터의 자동정렬 기능을 이용하면 편해요 !